airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -56,6 +56,19 @@ def parse_response(self, response: requests.Response, stream_state: Mapping[str,
                     record.pop(field, None)
                     yield record
         yield from records
+        
+    def backoff_time(self, response: requests.Response) -> Optional[float]:
+        """This method is called if we run into the rate limit.
+        Twilio puts the retry time in the `Retry-After` response header so we
+        we return that value. If the response is anything other than a 429 (e.g: 5XX)
+        fall back on default retry behavior.
+        Rate Limits Docs: https://support.twilio.com/hc/en-us/articles/360032845014-Verify-V2-Rate-Limiting"""
+
+        if "Retry-After" in response.headers:
+            return int(response.headers["Retry-After"])
+        else:
+            self.logger.info("Retry-after header not found. Using default backoff value")
+            return None
```

Comment:
Suggestion for conciness:
```suggestion
        backoff_time = response.headers.get("Retry-After")
        if backoff_time is not None:
          return float(backoff_time)
```
I suggest returning a float because this is the return type suggested by the parent method.

Reply:
sure! will check this and the tests 💪